TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/BasicMotorTest.java

```diff
@@ -10,35 +10,44 @@
 /**
  * TeleOp OpMode for testing motors.
  */
+// "BasicMotorTest" is the name of OpMode in Driver Station
```

Comment:
If you're going to mix JavaDoc comments and regular Java comments, consider putting the Java comments above the JavaDoc comments, like so:

```java
/* Java */
/** JavaDoc */
```

Not

```java
/** JavaDoc */
/* Java */
```